courier/template/load_template.go

```diff
@@ -105,23 +129,41 @@ func loadTemplate(osdir, name, pattern string, html bool) (Template, error) {
 	return tpl, nil
 }
 
-func LoadTextTemplate(osdir, name, pattern string, model interface{}) (string, error) {
-	t, err := loadTemplate(osdir, name, pattern, false)
+func LoadTextTemplate(osdir, name, pattern string, model interface{}, fs ...embed.FS) (string, error) {
+	var t Template
+	var err error
+
+	if len(fs) != 0 && osdir == "" {
+		t, err = loadTemplateFs(fs[0], name, pattern, false)
+	} else {
+		t, err = loadTemplate(osdir, name, pattern, false)
+	}
+
 	if err != nil {
 		return "", err
 	}
+
 	var b bytes.Buffer
 	if err := t.Execute(&b, model); err != nil {
 		return "", err
 	}
 	return b.String(), nil
 }
 
-func LoadHTMLTemplate(osdir, name, pattern string, model interface{}) (string, error) {
-	t, err := loadTemplate(osdir, name, pattern, true)
+func LoadHTMLTemplate(osdir, name, pattern string, model interface{}, fs ...embed.FS) (string, error) {
```

Comment:
It looks like we will only use the first embedFS argument. So maybe just make this a regular arg?

Reply:
but then it won't be optional anymore. we will then require the current functions to pass a nil i guess

Reply:
I see, okay, I will implement option 2.

Reply:
Thank you @Benehiko  :)